tasks/main.yml

```diff
@@ -25,3 +25,7 @@
     regexp: "^::1"
     line: "::1{{'\t\t'}}{{inventory_hostname}}{% if inventory_hostname != inventory_hostname_short %}{{'\t'}}{{inventory_hostname_short}}{% endif %}{{'\t'}}localhost ip6-localhost ip6-loopback"
     state: present
+
+- name: Hostname | Install Avahi-MDNS
+  apt: pkg=avahi-daemon state=latest
+  when: "'{{ansible_os_family}}' == 'Debian' and {{hostname_avahi}}"
```

Comment:
This doesn't have to be in a string, it can be:
`ansible_os_family == "Debian" and hostname_avahi`
:smile: